randovania/data/json_data/prime1.txt

```diff
@@ -196,207 +199,421 @@ Transport to Magmoor Caverns West
+      Trivial
+
+> Pickup (Spider Track Missile); Heals? False
+  * Pickup 36; Major Location? False
+  > Dock to Ice Ruins Access
+      Trivial
 
 ----------------
 Temple Entryway
 Asset id: 2245653401
 > Dock to Chozo Ice Temple; Heals? False
   * Normal Door to Chozo Ice Temple/Dock to Temple Entryway
+  > Dock to Phendrana Shorelines
+      Plasma Beam or Missile or Charge Beam
 
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Temple Entryway
+  > Dock to Chozo Ice Temple
+      Plasma Beam or Missile or Charge Beam
 
 ----------------
 Save Station B
 Asset id: 92367261
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Save Station B
+  > Save Station
+      Trivial
+
+> Save Station; Heals? True
+  > Dock to Phendrana Shorelines
+      Trivial
 
 ----------------
 Ruins Entryway
 Asset id: 1007903802
 > Dock to Ice Ruins West; Heals? False
   * Normal Door to Ice Ruins West/Dock to Ruins Entryway
+  > Dock to Phendrana Shorelines
+      Trivial
 
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Ruins Entryway
+  > Dock to Ice Ruins West
+      Trivial
 
 ----------------
 Plaza Walkway
 Asset id: 3356578450
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Plaza Walkway
+  > Dock to Ice Ruins East
+      Trivial
 
 > Dock to Ice Ruins East; Heals? False
   * Normal Door to Ice Ruins East/Dock to Plaza Walkway
+  > Dock to Phendrana Shorelines
+      Trivial
 
 ----------------
 Ice Ruins Access
 Asset id: 1317347388
 > Dock to Ice Ruins East; Heals? False
   * Normal Door to Ice Ruins East/Dock to Ice Ruins Access
+  > Dock to Phendrana Shorelines
+      Plasma Beam or Missile or Charge Beam
 
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Ice Ruins Access
+  > Dock to Ice Ruins East
+      Plasma Beam or Missile or Charge Beam
 
 ----------------
 Chozo Ice Temple
 Asset id: 1716909342
 > Dock to Temple Entryway; Heals? False
   * Normal Door to Temple Entryway/Dock to Chozo Ice Temple
+  > Pickup (Artifact of Sun)
+      Plasma Beam and Morph Ball Bomb and Space Jump Boots and Morph Ball
+  > Event (Chozo Ice Temple Bomb Slot)
+      Morph Ball Bomb and Space Jump Boots and Morph Ball
 
 > Dock to Chapel Tunnel; Heals? False
   * Normal Door to Chapel Tunnel/Dock to Chozo Ice Temple
+  > Dock to Temple Entryway
+      After Chozo Ice Temple Bomb Slot
+  > Pickup (Artifact of Sun)
+      Plasma Beam and Missile and Morph Ball Bomb and Morph Ball and After Chozo Ice Temple Bomb Slot
+
+> Pickup (Artifact of Sun); Heals? False
+  * Pickup 37; Major Location? True
+  > Dock to Temple Entryway
+      Morph Ball
+  > Dock to Chapel Tunnel
+      Morph Ball
+
+> Event (Chozo Ice Temple Bomb Slot); Heals? False
+  > Dock to Temple Entryway
+      Trivial
+  > Dock to Chapel Tunnel
+      Trivial
```

Comment:
For Chozo Ice Temple:
- Add a helper node called "Front of Bomb Slot." Connect the node to all the other nodes as follows:
  - Bomb Slot with Morph, Missile, Bombs
  - Artifact of Sun with Space Jump, Plasma, Morph
  - Chapel Tunnel with After Bomb Slot
  - Temple Entryway as Trivial
- The following nodes should only connect to the helper node:
  - Bomb Slot as Trivial
  - Chapel Tunnel with After Bomb Slot
  - Artifact of Sun as Trivial
  - Temple Entryway with Space Jump